### ISSUE TITLE
Youtube demo

### ISSUE THREAD

**Issue by ssured** [jerel/poc#1]
Hi Jerel, I just saw your youtube ember leaflet demo. Impressive stuff. Are you willing to share any insights in how you render the templates in the dialogs? It looks awesome! Thanks, Sjoerd

**Comment by jerel** [comment]
This is where the magic happens: https://github.com/jerel/poc/blob/master/app/components/leaflet-popup.js#L4-L8 I'm creating my own popup element which I then pass to Leaflet as the popup content and also pass to the _super render method so it adds the template to it

**Comment by ssured** [comment]
Thanks for responding so fast! Cool, I'll poke around with those lines. In the video it looks like you also added some hooks to the leaflet controls, did you? 
Did you also play around with the new HTMLBars in 1.10? The lines you reference seem to use private API, it might break? With the new HTMLBars I think it will finally be possible to write some nice leaflet-ember components. This pull https://github.com/emberjs/ember.js/pull/10244 seems quite promising too

**Comment by ssured** [comment]
For people coming here, it's all about this demo: https://t.co/ACEUZHK5Hw

**Comment by jerel** [comment]
> In the video it looks like you also added some hooks to the leaflet controls, did you?

No, I instead I implement my own controls and absolutely position them over the map. The tile switcher is as simple as `{{#radio-button value="street" groupValue=selectedTiles}}Streets{{/radio-button}}` which the tile component observes.

> Did you also play around with the new HTMLBars in 1.10? 

Yes, I run `master`

> The lines you reference seem to use private API, it might break?

Yes, no guarantees. However it's minimal, `buffer._element` and `renderer` IIRC
